packages/gestalt/src/Tabs.js

```diff
@@ -204,7 +204,7 @@ type Props = {|
    */
   onChange: OnChangeHandler,
   /**
-   *
+   * The array of tabs to be displayed. The active tab (as indicated by `activeTabIndex`) will be underlined. Use the [optional `indicator` field](https://gestalt.pinterest.systems/web/tabs#Indicator) to show a notification of new items on the tab. Though `text` currently accepts a React.Node, this is deprecated and will be replaced by a simple `string` type soon.
```

Comment:
> Though `text` currently accepts a React.Node, this is deprecated and will be replaced by a simple `string` type soon

I'm looking into this at some point — just trying to prevent additional usages for now

Reply:
@dangerismycat The underline looks funny here. Could we do a regular See variant X to learn more type of reference?

<img width="362" alt="Screen Shot 2022-09-09 at 2 29 17 PM" src="https://user-images.githubusercontent.com/10593890/189350147-a33b3b9e-acba-4e98-9cfe-322af50199e3.png">

Reply:
I was trying to avoid that to make the text flow better, but you're right, the underline+code is odd.